README.md

```diff
@@ -31,10 +31,17 @@ const client = createClient({
     amqps: true,
     frameMax: 8192,
   },
+  queueOptions: {
+    args: {
```

Comment:
why such additional key `args` is needed?

Reply:
In order to have a dedicated args object, just for future support of other options, not to mix them up with the args group

Reply:
but you can support new options in `queueOptions` directly. You do not need any additional wrapper such as `args`